exectors/nodejs/collator.js

```diff
@@ -0,0 +1,32 @@
+// The Collator used for the actual testing.
+
+// !!! TODO: Collation: determine the sensitivity that corresponds
+// to the strength.
+module.exports = {
+
+  testCollationShort: function(json) {
+
+    // Global default locale
+    let testLocale = '';
+    let testCollOptions = {ignorePunctuation:'true'};
+
+    // Set up collator object with optional locale and testOptions.
+    let coll;
```

Comment:
As best practice: initialize with 0/null.

Reply:
This is OK in JS because it sets to "undefined".